pkgs/top-level/haskell-packages.nix

```diff
@@ -134,6 +134,15 @@ in
         inherit buildTargetLlvmPackages llvmPackages;
       };
       ghc912 = compiler.ghc9122;
+      ghc9141 = callPackage ../development/compilers/ghc/9.14.1.nix {
+        bootPkgs =
+          # No suitable bindist packaged yet
+          bb.packages.ghc9103;
```

Comment:
`ghcHEAD` below works with `bootPkgs = bb.packges.ghc984Binary`. I'd expect 9.14 to work the same way (but I'm just guessing), which is a shorter bootstrap path than going via 9.10.